regulation-worker/cmd/main.go

```diff
@@ -80,9 +80,10 @@ func Run(ctx context.Context) {
 				FilesLimit: config.GetInt("REGULATION_WORKER_FILES_LIMIT", 1000),
 			},
 			&api.APIManager{
-				Client:           &http.Client{Timeout: config.GetDuration("HttpClient.regulationWorker.transformer.timeout", 60, time.Second)},
-				DestTransformURL: config.MustGetString("DEST_TRANSFORM_URL"),
-				OAuth:            OAuth,
+				Client:                       &http.Client{Timeout: config.GetDuration("HttpClient.regulationWorker.transformer.timeout", 60, time.Second)},
+				DestTransformURL:             config.MustGetString("DEST_TRANSFORM_URL"),
+				OAuth:                        OAuth,
+				MaxOAuthRefreshRetryAttempts: config.GetInt("RegulationWorker.oauth.maxRefreshRetryAttempts", 1),
```

Comment:
I'd not tie this variable to OAuth as it's a delete retry we are doing rather than a "refresh retry".
Moreover, we might retry a delete request due to other failure scenarios that could possibly succeed in the next retry.

Reply:
The retry only is required for the OAuth case in order to prevent the case where in a failed job is retried after the token's validity period.